timml/well.py

```diff
@@ -301,7 +301,7 @@ def plotcapzone(
 
 
 class Well(WellBase, MscreenWellEquation):
-    """Well Class to create a well with a specified discharge.
+    r"""Well Class to create a well with a specified discharge.
```

Comment:
What does this r do here?

Reply:
There's TeX in the docstring:

```    .. math::
        Q_i = 2\pi r_w(h_i - h_w)/c
```

Ruff says the r needs to be prepended in case the docstring contains backslashes (basically to escape them as a raw string).

Reply:
Fair enough, I didnt know that and the docs look good :+1: